src/index.html

```diff
@@ -1,13 +1,564 @@
+                  </a>
+
+                  <div class="row-news__subtitle">
+                    9 серпня 2019
+                  </div>
+
+                  <a class="row-news__title" href="#" target="_blank">
+                    Оголошення переможця
+                  </a>
+
+                  <p class="row-news__text">
+                    Друзі, сьогодні п'ятниця! А це означає, що час оголосити переможця розіграшу...
+                  </p>
+                </article>
+              </div>
+
+              <div class="row-news__column">
+                <article class="row-news__item">
+                  <a href="#" class="row-news__image _ibg">
+                    <img src="img/news/news_02.webp" alt="cats">
+                  </a>
+
+                  <div class="row-news__subtitle">
+                    9 серпня 2019
+                  </div>
+
+                  <a class="row-news__title" href="#" target="_blank">
+                    Міжнародний день котів
+                  </a>
+
+                  <p class="row-news__text">
+                    Музей з левами не може просто так взяти і пропустити Міжнародний день котів!
+                  </p>
+                </article>
+              </div>
+
+              <div class="row-news__column">
+                <article class="row-news__item">
+                  <a href="#" class="row-news__image _ibg">
+                    <img src="img/news/news_03.webp" alt="kulish">
+                  </a>
+
+                  <div class="row-news__subtitle">
+                    9 серпня 2019
+                  </div>
+
+                  <a class="row-news__title" href="#" target="_blank">
+                    200 років П. Кулішу
+                  </a>
+
+                  <p class="row-news__text">
+                    Поруч із Тарасом Шевченком це найважливіша постать в українській культурі ХІХ
+                    століття...
+                  </p>
+                </article>
+              </div>
+            </div>
+
+            <div class="news__all bottom__block">
+              <a class="bottom__block__link" href="">Усі новини</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__subscribe subscribe" id="subscribe">
+        <div class="subscribe__container _container">
+          <div class="subscribe__body">
+            <h2 class="subscribe__title title">
+              Підпишіться на дайджест
+            </h2>
+
+            <div class="subscribe__row">
+              <p class="subscribe__text">Першими дізнавайтесь про новини музею та розіграші, отримуйте
+                запрошення на
+                події та читайте статті від кураторів
+              </p>
+
+              <form action="#" class="subscribe__form form-subscribe">
+                <input class="form-subscribe__input input" autocomplete="off" type="email" name="email"
+                  placeholder="e-mail" required>

```

Comment:
Fix code style for all code

```suggestion
                <input
                  class="form-subscribe__input input"
                  autocomplete="off"
                  type="email"
                  name="email"
                  placeholder="e-mail"
                  required
                >

```